Library/Formula/proftpd.rb

```diff
@@ -1,8 +1,8 @@
 class Proftpd < Formula
   desc "Highly configurable GPL-licensed FTP server software"
   homepage "http://www.proftpd.org/"
-  url "ftp://ftp.proftpd.org/distrib/source/proftpd-1.3.4d.tar.gz"
-  sha256 "c198b53991ce641eae6b3237e856e59f0bfe8330794145b49cae33f85b6f5370"
+  url "ftp://ftp.proftpd.org/distrib/source/proftpd-1.3.5a.tar.gz"
+  sha256 "a1f48df8539c414ec56e0cea63dcf4b8e16e606c05f10156f030a4a67fae5696"
 
   # fixes unknown group 'nogroup'
   patch :DATA
```

Comment:
http://www.proftpd.org/docs/faq/linked/faq-ch4.html#AEN434
> The default ProFTPD configuration file uses the user "nouser" and the group "nogroup", some systems / distributions do not have the group "nogroup" defined. The solution is to either add the group "nogroup" to /etc/groups or to change the "nogroup" entry in the proftpd.conf to a group which does exist.

Would it be agreeable to change the patch into an `inreplace` line?